pkg/tracker/currentVariables.go

```diff
@@ -5,29 +5,34 @@ package tracker
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 	"github.com/pkg/errors"
 	tellorCommon "github.com/tellor-io/TellorMiner/pkg/common"
 	"github.com/tellor-io/TellorMiner/pkg/config"
 	"github.com/tellor-io/TellorMiner/pkg/contracts/getter"
 	"github.com/tellor-io/TellorMiner/pkg/db"
-	"github.com/tellor-io/TellorMiner/pkg/util"
 )
 
-var currentVarsLog = util.NewLogger("tracker", "CurrentVarsTracker")
-
 const CurrentVariablesTrackerName = "CurrentVariablesTracker"
 
 type CurrentVariablesTracker struct {
+	logger log.Logger
 }
 
 func (b *CurrentVariablesTracker) String() string {
 	return CurrentVariablesTrackerName
 }
 
+func New_CurrentVariablesTracker(logger log.Logger) *CurrentVariablesTracker {
```

Comment:
```suggestion
func NewCurrentVariablesTracker(logger log.Logger) *CurrentVariablesTracker {
```

Reply:
This one is a hack because we already have another tracker call `NewCurrentVariables`, so this is the least awkward way of solving that. But I'm open to suggestions!

Reply:
aah yeah this the v1 code. The `pkg/tracker/currentVariables.go` file can be removed and this will resolve the conflict. If you prefer can do it in a parallel PR or can do it even in this one.

Reply:
I adjusted the naming but left the tests commented. I'll adjust those in a next PR